src/service/standard-actions-impl.js

```
@@ -125,6 +131,21 @@ export class StandardActions {
    * @param {!./action-impl.ActionInvocation} invocation
    * @private
    */
+  handleAmpNavigateTo_(invocation) {
+    if (!invocation.satisfiesTrust(ActionTrust.HIGH)) {
+      return;
+    }
+    const url = invocation.args['url'];
```

Would an error message here be better, or would you prefer it silently fail?